### ISSUE TITLE
Rebuild failed on OSX probably after command line tools upgrade

### ISSUE THREAD

**Issue by mauron85** [TypeStrong/atom-typescript#597]
Hi, today Atom showed following message:
*Some installed packages could not be loaded because they contain native modules that were compiled for an earlier version of Atom.*

I clicked **REBUILD PACKAGES** but build failed. I think it has something to do with latest command line tools for OSX.

OSX: 10.10.5
Atom: 1.0.15
atom-typescript: 6.0.9
Command Line Tools (OSX 10.10) for XCode: 7.0 (September 19, 2015)

Log:
```
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

In file included from ../fsevents.cc:6:
In file included from ../node_modules/nan/nan.h:74:
In file included from ../node_modules/nan/nan_new.h:181:
../node_modules/nan/nan_implementation_12_inl.h:172:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/finch/.atom/.node-gyp/.node-gyp/0.30.6/deps/v8/include/v8.h:4515:3: note: 'New' declared here
  static Local<Signature> New(
  ^
1 error generated.
make: *** [Release/obj.target/fse/fsevents.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/finch/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "node" "/Users/finch/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/finch/.atom/packages/atom-typescript/node_modules/grunt-ts/node_modules/chokidar/node_modules/fsevents
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v2.0.2
gyp ERR! not ok 

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/finch/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Users/finch/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/Users/finch/.atom/.apm/.apmrc" "--userconfig" "/Users/finch/.atom/.apmrc" "rebuild" "--target=0.30.6" "--arch=x64"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE
npm ERR! fsevents@0.3.5 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fsevents@0.3.5 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the fsevents package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls fsevents
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/finch/.atom/packages/atom-typescript/npm-debug.log
```

**Comment by ralfbiedert** [comment]
Same problem here, although I get the message also for 6.0.8

<img width="1006" alt="screen shot 2015-09-21 at 10 51 29" src="https://cloud.githubusercontent.com/assets/301288/9988199/c1618300-604e-11e5-9cd9-add1b7eaa991.png">

**Comment by ralfbiedert** [comment]
Update: Looks like I was able to fix it after manually updating to 6.0.13.

**Comment by mauron85** [comment]
Ok. Seem fixed in 6.0.13.

**Comment by basarat** [comment]
> Ok. Seem fixed in 6.0.13.

Basically an atom bug https://github.com/atom/atom/issues/8837 that's been fixed :rose:
